packages/sources/svelte/package.json

```diff
@@ -64,20 +67,18 @@
     "@vtmn/css-toast": "^0.7.2",
     "@vtmn/css-toggle": "^0.8.2",
     "@vtmn/css-tooltip": "^0.8.2",
-    "@vtmn/icons": "^0.14.0"
-  },
-  "devDependencies": {
-    "@rollup/plugin-node-resolve": "^13.1.2",
-    "@testing-library/jest-dom": "^5.16.2",
-    "@testing-library/svelte": "^3.0.3",
+    "@vtmn/icons": "^0.14.0",
```

Comment:
We could leave the `@vtmn` packages as dependencies, it could be a breaking changes if the `/src` folder is still used.

Reply:
Not necessary because the dist deliver all the needed sources.
In fact, when we compile the source code, now, we getting the sources from the dist folder.

But yes, of course, it can be a breaking change if we use again the `/src` folder

Reply:
If the `/src` folder is still available and accessible, we should leave them as dependencies.

Like said by NPM:
```
Development dependencies are intended as development-only packages, that are unneeded in production. For example testing packages, webpack or Babel.
```
https://nodejs.dev/en/learn/npm-dependencies-and-devdependencies
https://docs.npmjs.com/specifying-dependencies-and-devdependencies-in-a-package-json-file